packages/arb-bridge-peripherals/contracts/tokenbridge/libraries/ERC165.sol

```diff
@@ -0,0 +1,32 @@
+// SPDX-License-Identifier: MIT
+// OpenZeppelin Contracts v4.4.1 (utils/introspection/ERC165.sol)
+
+// With modification to Solidity ^0.6.11
+// https://github.com/OpenZeppelin/openzeppelin-contracts/blob/release-v4.6/contracts/utils/introspection/ERC165.sol
```

Comment:
why not use the previous OZ release instead?
https://github.com/OpenZeppelin/openzeppelin-contracts/blob/v3.4.2/contracts/introspection/ERC165.sol

Reply:
> pragma solidity >=0.6.0 <0.8.0;
it doesn't support ^0.8.0 which break our interface compatibility anyway.

Reply:
hm yea it also has a different implementation. 4.6 uses the solidity interface Id instead of keeping mappings

Reply:
added some comments to explain